code/modules/modular_computers/networking/machinery/telecomms.dm

```diff
@@ -330,6 +331,25 @@ var/global/list/telecomms_hubs = list()
 									choice = input(user, "Which key do you wish to remove?", "Channel Configuration") as null|anything in channel_datum.secured
 									if(choice && CanPhysicallyInteract(user))
 										LAZYREMOVE(channel_datum.secured, choice)
+							if("Add Network Group")
+								var/datum/extension/network_device/network_device = get_extension(src, /datum/extension/network_device)
+								var/datum/computer_network/network = network_device?.get_network()
+								if(!network)
+									to_chat(user, SPAN_WARNING("Unable to connect to the network."))
+									return TOPIC_HANDLED
+
+								var/datum/extension/network_device/acl/net_acl = network.access_controller
+								if(!net_acl)
+									to_chat(user, SPAN_WARNING("No access controller on the network."))
```

Comment:
No return here? It would runtim hen getting the groups

Reply:
Good catch, fixed